woocommerce/woocommerce-ios
# [Woo POS] Revert WooCommercePOS framework

Closes: #12745 
Based on https://github.com/woocommerce/woocommerce-ios/pull/12740 to include all the POS files


## Why

As announced in the sync meeting, we're reverting the addition of the POS framework in https://github.com/woocommerce/woocommerce-ios/pull/12677. This PR moves all the files from the POS framework to the main app in a POS folder.

## How

Just two structs need to be renamed in https://github.com/woocommerce/woocommerce-ios/pull/12747/commits/57fbc2b6a4bedfc9330b833151681bf89f7c907e, `ProductFactory` -> `POSProductFactory` and `Product` -> `POSProduct` (lemme know if you think `PointOfSale` prefix is a better choice). I don't think it makes sense to rename all POS files with a `POS` prefix, in case we ever have plans to move them to a separate framework/app. The rest of the changes are just deleted files from the removal of the WooCommercePOS framework.

## Testing instructions


Feel free to do a confidence check on the POS entry point:

- Go to the Menu tab
- Ensure the POS feature switch is enabled (if not, go to Settings > Experimental Features, then enable "Point of Sale")
- Tap `Point of Sale` menu item --> the product cart UX, reader connection UX should stay the same

---
